apps/bfd-pipeline/bfd-pipeline-rda-grpc-server/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/StoreRdaJsonApp.java

```diff
@@ -68,21 +68,23 @@ public static void main(String[] args) throws Exception {
     }
   }
 
-  private static ManagedChannel createChannel(String host, int port) {
-    final ManagedChannelBuilder<?> channelBuilder = ManagedChannelBuilder.forAddress(host, port);
-    if (host.equals("localhost")) {
+  private static ManagedChannel createChannel(Config config) {
+    final ManagedChannelBuilder<?> channelBuilder =
+        ManagedChannelBuilder.forAddress(config.grpcConfig.getHost(), config.grpcConfig.getPort());
+    if (config.grpcConfig.getHost().equals("localhost")) {
```

Comment:
what about 127.0.0.1?  Maybe `InetAddress.getByName(config.grpcConfig.getHost()).isAnyLocalAddress()`?

Reply:
That's a good enhancement that we could apply in a few places in a later PR.